src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -58,18 +58,9 @@
         <span
           >Trades from ETH route through Balancer liquidity pools and incur gas
           fees.
-          <router-link
-            :to="{
-              name: 'trade',
-              params: {
-                assetIn: TOKENS.AddressMap.WETH,
-                assetOut: trading.tokenOut.value.address
-              }
-            }"
-            class="text-blue-500"
-          >
+          <a @click="switchToWETH()" class="text-blue-500">
             Trade from WETH
-          </router-link>
```

Comment:
I've a made a little mistake here - it should switch to ETH "softly" without route change - otherwise the input value is lost.